ui/src/components/group/AddGroup.js

```diff
@@ -317,13 +350,26 @@ class AddGroup extends React.Component {
                         </ContentDiv>
                     </SectionDiv>
                 }
-                <SectionDiv>
+                <SectionDiv title={auditToolTip}>
                     <StyledInputLabel />
                     <StyledIncludedMembersDiv>
                         {members}
                     </StyledIncludedMembersDiv>
                 </SectionDiv>
                 {this.getJustification()}
```

Comment:
in the function there is no need to use justificationRequired instead use isDomainAuditEnabled and stop passing it from GroupList

Reply:
i cleaned it